src/server/worldserver/worldserver.conf.dist

```diff
@@ -2167,7 +2167,7 @@ GM.LogTrade = 1
 #        Description: GM character starting level.
 #        Default:     1
 
-GM.StartLevel = 80
+GM.StartLevel = 1
```

Comment:
I think it's handy to have it at `80` for GMs. Maybe we should just fix the `Default` ?

Reply:
Personally I hate it being default 80 for GMs, just because for me it makes no sense to default force 80 on a GM accounts :shrug:

Reply:
ok, fair enough :)